app/__main__.py

```diff
@@ -0,0 +1,25 @@
+"""Full Translation App"""
+
+from os import getenv
+from pyrogram import Client
+from consumers import TelegramConsumer
+from translators import PythonTranslator
+from app import TranslationApp
+
+api_id = getenv('API_ID')

```

Comment:
Add error handling around if these values are None; perhaps print a descriptive log of the missing env var and then sys.exit